Essentials/src/com/earth2me/essentials/commands/Commandmute.java

```diff
@@ -40,20 +41,29 @@ public void run(final Server server, final CommandSource sender, final String co
                 throw new Exception(tl("muteExempt"));
             }
         }
+
+        long muteTimestamp = 0;
+        String time;
+
+        if (args.length > 1) {
+            time = args[1];
+            muteTimestamp = DateUtil.parseDateDiff(time, true);
+            final long maxMuteLength = ess.getSettings().getMaxMute() * 1000;
```

Comment:
I would maybe prefer this conversion to happen in settings. That or note the units as a comment in the settings file - standalone it is unclear what the units are.

Reply:
It's noted in config.yml which is what users of the plugin will be mostly concerned with. That line of code is borrowed from Commandtempban.java, so if there's a problem with following this code convention that should probably be looked at.

Reply:
This is a documentation nit: I didn't know immediately from looking at it so I brought it up.

Just because it is copied doesn't mean it is right

Reply:
I agree with you, but let's do that for both in a separate refactor PR. I think it would be ideal for usage to stay consistent all the way through. If conversions are to be moved into settings, I think that should be done with all other outstanding commands as well.